freshman/dApp.md

```diff
@@ -1 +1,6 @@
 # dApp
+
+### Question 1: 
+- I was able to set the mood but can't seem to get the mood what am i doing wrong?
```

Comment:
the formatting is wrong, check how I formatted this in the rest of the files. You need to make the whole question bold